site2/docs/administration-load-balance.md

```diff
@@ -6,96 +6,74 @@ sidebar_label: Load balance
 
 In case of partitioned topics, different partitions are assigned to different brokers. Here "topic" means either a non-partitioned topic or one partition of a topic.
 
-The assignment is "dynamic" because it changes quickly. For example, if the broker owning
-the topic crashes, the topic will be reassigned immediately to another broker. Another scenario is
-that the broker owning the topic becomes overloaded. In this case, the topic is
-reassigned to a less loaded broker.
+The assignment is "dynamic" because the assignment changes quickly. For example, if the broker owning the topic crashes, the topic is reassigned immediately to another broker. Another scenario is that the broker owning the topic becomes overloaded. In this case, the topic is reassigned to a less loaded broker.
 
-The dynamic assignment is made possible by the stateless nature of brokers, so you can 
-quickly expand or shrink the cluster based on usage.
+The stateless nature of brokers makes the dynamic assignment possible, so you can quickly expand or shrink the cluster based on usage.
 
-### Assignment granularity
+#### Assignment granularity
 
-The assignment of topics/partitions to brokers is not done at the topics/partitions level, but done at the Bundle level (a higher level). The reason is to amortize the amount of information that you need to keep track. Based on CPU, memory, traffic load and other indexes, topics are assigned to a particular broker dynamically. 
+The assignment of topics or partitions to brokers is not done at the topics or partitions level, but done at the Bundle level (a higher level). The reason is to amortize the amount of information that you need to keep track. Based on CPU, memory, traffic load and other indexes, topics are assigned to a particular broker dynamically. 
 
-Instead of individual topic/partition assignment, each broker takes ownership of a subset of the
-topics for a namespace. This subset is called a "*bundle*" and effectively it's a sharding
-mechanism.
+Instead of individual topic or partition assignment, each broker takes ownership of a subset of the topics for a namespace. This subset is called a "*bundle*" and effectively this subset is a sharding mechanism.
 
-The namespace is the "administrative" unit: many config knobs or operations are done at the
-namespace level.
+The namespace is the "administrative" unit: many config knobs or operations are done at the namespace level.
 
 For assignment, a namespaces is sharded into a list of "bundles", with each bundle comprising
 a portion of overall hash range of the namespace.
 
-Topics are assigned to a particular bundle by taking the hash of the topic name and seeing in which
-bundle the hash falls into.
+You can take the hash of the topic name and see in which bundle the hash falls into to assign topics to a particular bundle.
```

Comment:
Using "you" here makes it sounds as an operation that will be done by a user/administrator, while in fact this is done by brokers.